packages/@aws-cdk/aws-apigateway/lib/restapi.ts

```diff
@@ -159,6 +159,14 @@ export interface RestApiBaseProps {
    * @default - when no export name is given, output will be created without export
    */
   readonly endpointExportName?: string;
+
+  /**
+   * A list of the endpoint types of the API. Use this property when creating
+   * an API.
+   *
+   * @default - No endpoint types that translates to a EndpointType.EDGE endpoint type.
```

Comment:
```suggestion
   * @default EndpointType.EDGE
```

Reply:
👍  I will apply that to the other messages also

Reply:
done as with the other ones in 44618d9